app/src/main/java/com/example/alexandraboukhvalova/a436msclinicalmonitoring/MainActivity.java

```diff
@@ -11,6 +12,9 @@
 import android.view.Menu;
 import android.view.MenuItem;
 import android.widget.Button;
+import android.os.*;
```

Comment:
Not an issue for now, but using the Wildcard on Java imports can be risky for clashing class names (i.e. a class "List" exists in both java.util.* and java.awt.* then stuff starts breaking). Just something to be on the lookout for in the future.